public/kibi_timeline.js

```diff
@@ -80,6 +81,19 @@ define(function (require) {
         if (!timeline) {
           // create a new one
           timeline = new vis.Timeline($element[0]);
+          var timezone = config.get('dateFormat:tz');
+          if (timezone !== 'Browser') {
+            var utcOffset = null;
+            if (timezone.substring(0,7) === 'Etc/GMT') {
```

Comment:
With **"Etc/GMT"**? Yes, I tried with these time zones (the _"Etc/GMT category"_) and I had to modify my function because it didn't remove hours (in _"Etc/GMT**-**[hours]"_ use case, otherwise, in _"Etc/GMT**+**[hours]"_ it works)